x-pack/legacy/plugins/epm/server/registry/index.ts

```diff
@@ -87,7 +88,7 @@ export function pathParts(path: string): AssetParts {
   }
 
   // This is to cover for the fields.yml files inside the "fields" directory
-  if (file === undefined && type.endsWith('.yml')) {
+  if (file === undefined) {
```

Comment:
```suggestion
  if (file === undefined && type.endsWith('.yml')) {
```

Reply:
@jfsiii This does not work as expected and currently breaks feature-ingest branch :-(

Reply:
😬 ok. Whatever you need to get it working. We can come back to this.

Reply:
We will have to come back to it, it's ugly 🤦‍♂